web-ui/github.ml

```diff
@@ -310,35 +310,61 @@ let repo_handle ~meth ~owner ~name ~repo path =
           let variant = job_i.variant in
           Capability.with_ref (Client.Commit.job_of_variant commit variant) @@ fun job ->
           Current_rpc.Job.status job |> Lwt.map (function
-              | Ok s -> if s.Current_rpc.Job.can_cancel then Some job else None
+              | Ok s -> if s.Current_rpc.Job.can_cancel then Some (job_i, job) else None
               | Error _e -> None
```

Comment:
```suggestion
              | Error e -> Log.info (fun f -> f "Error fetching job status: %a" Capnp_rpc.Error.pp e); None
```

Print the error as an `info` in case it's useful.